tests/bsim/bluetooth/audio/src/cap_initiator_unicast_test.c

```diff
@@ -694,6 +696,7 @@ static void unicast_audio_stop(struct bt_bap_unicast_group *unicast_group)
 
 	/* Stop without release first to verify that we enter the QoS Configured state */
 	UNSET_FLAG(flag_stopped);
+	printk("Stopping without relasing\n");
```

Comment:
I assume you intended to keep this and the other printk statements

Reply:
Indeed. They are useful for debugging purposes